.tsan-suppressions

```diff
@@ -0,0 +1,4 @@
+race:gloo::transport::tcp::Pair::read
```

Comment:
do we know why thread sanitizer thinks there are races here?

Reply:
Yes, this is caused by the multi threaded tests where a send may not be protected by a mutex, whereas a read (triggered from the I/O thread) is always protected by a mutex.